### PR TITLE
1.4.1 Release Notes

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Comment by howardjohn** [comment]
Linter failure is expected, as we have not shipped 1.4.1 yet. What is the right action here? There have been complaints from our users about publishing docs after the release is shipped, so it seems like we want to ignore the lint errors rather than push the docs after we ship

File: content/en/news/releases/1.4.x/announcing-1.4.1/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.4.1
+linktitle: 1.4.1
+subtitle: Patch Release
+description: Istio 1.4.1 patch release.
+publishdate: 2019-12-05
+release: 1.4.1
+aliases:
+    - /news/2019/announcing-1.4.1
```
Comment (geeknoid) [inline content/en/news/releases/1.4.x/announcing-1.4.1/index.md]: No need for this first alias.

**Comment by geeknoid** [comment]
You can force push this PR in right before publishing the release. But I don't think it's worth it.
This is a no-win situation. If you push the doc PR in first, folks that are listening to the RSS feed will get notified and not be able to download the release until it is published.

The only thing that works well is to line things up so there's only seconds of delay between one thing being published vs. the other.

**Comment by geeknoid** [comment]
When publishing release notes, you also need to update the full_version value in data/args.yaml

**Comment by howardjohn** [comment]
> When publishing release notes, you also need to update the full_version value in data/args.yaml

Right now its 1.5.0, and this is going to master. So do I manually cherrypick to 1.4 branch and set it there? Or should I have targetted this at 1.4 to begin with?

**Comment by geeknoid** [comment]
You'll need to do a manual cherry pick and change the version number in data/args.yml explicitly during the cherry pick.

**Comment by geeknoid** [comment]
Actually, you can do an automatic cherry pick, and then just do another PR to update the version separately.

**Comment by geeknoid** [comment]
Note that this PR also has a linter error due to a spelling issue.

File: content/en/news/releases/1.4.x/announcing-1.4.1/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** `istioctl` installation on Windows ([Issue 19020](https://github.com/istio/istio/pull/19020)).
+- **Fixed** an issue with route matching order when using CertManager with Kubernetes Ingress ([Issue 19000](https://github.com/istio/istio/pull/19000)).
+- **Fixed** Mixer source namespace attribute when the pod name contains a period ([Issue 19015](https://github.com/istio/istio/issues/19015)).
+- **Fixed** excessive metrics generated by Galley ([Issue 19165](https://github.com/istio/istio/issues/19165)).
+- **Fixed** tracing Service port to correctly listen on port 80 ([Issue 19227](https://github.com/istio/istio/issues/19227)).
+
```
Comment (ericvn) [inline content/en/news/releases/1.4.x/announcing-1.4.1/index.md]:
```suggestion
- **Fixed** istioctl.bash and _istioctl auto completion files missing ([Issue 19297](https://github.com/istio/istio/issues/19297)).
```

**Comment by howardjohn** [comment]
@istio/wg-docs-maintainers-english ptal

**Comment by frankbu** [comment]
Are you going to do a manual cherry-pick to release-1.4 with the data/args.yaml change? If so, we can approve both PRs, and they both can merge as soon as the broken release link is published. Does that make sense?

**Comment by howardjohn** [comment]
> Are you going to do a manual cherry-pick to release-1.4 with the data/args.yaml change? If so, we can approve both PRs, and they both can merge as soon as the broken release link is published. Does that make sense?

@frankbu that was the plan, just wanted to get this approved first so I don't have to try to keep two PRs in sync. I'll send a PR in a minute

**Comment by howardjohn** [comment]
https://github.com/istio/istio.io/pull/5979

**Pull request merged**